### DISCUSSION TITLE
Traffic generation

### DISCUSSION THREAD

**Discussion by alexj0l** [microsoft/msquic#2831] [Q&A] [answered]
Hi community,

I would like to know if there's any way to generate some traffic between a client and a server so that I can perf it and get some numbers for throughput etc?

Thanks in advance!

**Comment by nibanks** [comment, marked as answer]
This is similar to #2216, and I still have #2217 open to add documentation on how to run our performance tools. Also related is this small spec that we wrote and implement: https://datatracker.ietf.org/doc/html/draft-banks-quic-performance.
